docs/pages/guides/_meta.js

```diff
@@ -6,5 +6,6 @@ export default {
   "modules": "Writing MUD Modules",
   emojimon: "Emojimon",
   testing: "Testing",
+  dictionary: "MUD for Solidity Programmers",
```

Comment:
since this isn't a list of terms + definitions, can we call this something other than dictionary?
```suggestion
  solidity-patterns: "MUD for Solidity Programmers",
```